ui/raidboss/data/07-dt/dungeon/skydeep-cenote.txt

```diff
@@ -0,0 +1,244 @@
+
+# IGNORED ABILITIES
+# 8E5B Thunderlight Burst (boss-targeted)
+# 96B5 Thunderlight Burst (seequential cleave)
+# 96B6 Thunderlight Burst (seequential cleave)
+# 96B7 Thunderlight Burst (seequential cleave)
+# 96B8 Thunderlight Burst (seequential cleave)
+# 9705 Artillery (concurrent with 9704)
+# 9706 Artillery (concurrent with 9704)
+# 9707 Artillery (concurrent with 9704)
+
+# ALL ENCOUNTER ABILITIES
+# 368 attack
+# 8E5A Ancient Artillery
+# 8E5B Thunderlight Burst
+# 8E5D Thunderlight Burst
+# 8E5F Pummel
+# 8E60 Dynamic Dominance
+# 8E62 Thunderlight Flurry
+# 8E63 --sync--
+# 96B5 Thunderlight Burst
+# 96B6 Thunderlight Burst
+# 96B7 Thunderlight Burst
+# 96B8 Thunderlight Burst
+# 9704 Artillery
+# 9705 Artillery
+# 9706 Artillery
+# 9707 Artillery
+# 9858 Emergent Artillery
+# 98E3 Mirror Maneuver
+
+#~~~~~~~~~~~#
+# Maulskull #
+#~~~~~~~~~~~#
+
+# -ii 8F46 8F40 8F43 8F44 8F64 8F42 8F50 8F4E 8F53 8F54 8F3A 98D1 8F60 8F61 8F67 8F51
+# -p 8F3D:3013.1
+
+# Gatekeep's Anvil will be sealed off
+3000.0 "--sync--" SystemLogMessage { id: "7DC", param1: "13A9" } window 3000,1
+3005.2 "--sync--" StartsUsing { id: "8F3D", source: "Maulskull" }
+3013.1 "--sync--" Ability { id: "8F3D", source: "Maulskull" }
+3014.1 "Stonecarver (first)" Ability { id: "8F3E", source: "Maulskull" }
+3016.7 "Stonecarver (second)" #Ability { id: "8F3F", source: "Maulskull" }
+3030.4 "Impact + Skullcrush" Ability { id: "8F45", source: "Maulskull" }
+3033.0 "--sync--" Ability { id: "9708", source: "Maulskull" }
+3035.0 "Destructive Heat" Ability { id: "8F65", source: "Maulskull" }
+3044.3 "Maulwork" Ability { id: ["8F47", "8F48", "8F49", "8F4A"], source: "Maulskull" }
+3049.4 "Landing 1" Ability { id: "8F4B", source: "Maulskull" }
+3053.4 "Landing 2" Ability { id: "8F4B", source: "Maulskull" }
+3057.4 "Landing 3" Ability { id: "8F4B", source: "Maulskull" }
+3059.4 "Shatter" Ability { id: ["8F4C", "8F4D"], source: "Maulskull" }
+3074.4 "--sync--" Ability { id: "8F4F", source: "Maulskull" } # Deep Thunder cast
+3075.5 "Deep Thunder 1" #Ability { id: "8F52", source: "Maulskull" }
+3076.6 "Deep Thunder 2" #Ability { id: "8F52", source: "Maulskull" }
+3077.6 "Deep Thunder 3" #Ability { id: "8F52", source: "Maulskull" }
+3087.0 "Ringing Blows" Ability { id: ["8F56", "8F57"], source: "Maulskull" }
+3088.9 "Impact + Skullcrush" Ability { id: "8F3B", source: "Maulskull" }
+3091.0 "Stonecarver (first)" Ability { id: "8F58", source: "Maulskull" }
+3093.6 "Stonecarver (second)" Ability { id: "8F59", source: "Maulskull" }
+3102.3 "Wrought Fire" Ability { id: "98D2", source: "Maulskull" }
+3115.5 "Impact + Skullcrush" Ability { id: "8F63", source: "Maulskull" }
+3119.5 "Destructive Heat/Building Heat" Ability { id: ["8F65", "8F66"], source: "Maulskull" }
+3129.7 "Impact + Skullcrush" Ability { id: "8F63", source: "Maulskull" }
+3133.5 "Destructive Heat/Building Heat" Ability { id: ["8F65", "8F66"], source: "Maulskull" }
```

Comment:
Here and elsewhere, are these Heat usages always different, or have you seen an instance of 2x stack or spread in a row? If they are always different but in a random order, the second one should have the order reversed as a bit of a visual signal for that.

Reply:
I've only ever seen them alternate, with the first one being random.

Reply:
Same & can do.